cloudferrylib/os/identity/keystone.py

```diff
@@ -416,12 +432,11 @@ def _get_user_passwords(self):
 
         return info
 
-    def _get_user_tenants_roles(self):
+    def _get_user_tenants_roles(self, tenant_list=[], user_list=[]):
```

Comment:
I would not recommend use mutable default arguments.
http://docs.python-guide.org/en/latest/writing/gotchas/#mutable-default-arguments